hysr/rewards.py

```diff
@@ -0,0 +1,243 @@
+import math
+from .types import *
```

Comment:
If possible, try to avoid `from x import *` as this causes problems for linting tools.

Reply:
indeed ! fixed